styles/widgets/material/popup.material.less

```diff
@@ -27,7 +27,7 @@
 @MATERIAL_POPUP_TITLE_BOTTOM_MARGIN: @MATERIAL_POPUP_CONTENT_WITHTITLE_TOP_PADDING - @MATERIAL_POPUP_CONTENT_PADDING;
 
 .dx-popup-wrapper {
-    & > .dx-overlay-content {
+    &.dx-overlay-wrapper > .dx-overlay-content {
```

Comment:
This change brokes dxTooltip. If we need to change the tooltip background color in scheduler we need to use scheduler-specific classes.